### DISCUSSION TITLE
Zwift Hub Autoresistance

### DISCUSSION THREAD

**Discussion by bdattilo** [cagnulein/qdomyos-zwift#2112] [Ideas]
Hello -

Is auto resistance possible for Peloton workouts using the Zwift Hub? I see that the hub does read power and cadence correctly, but does not seem to be changing the resistance. 

I see that the [Kickr line is supported for auto resistance](https://robertoviola.cloud/2023/02/23/wahoo-kickr-finally-supported-on-peloton-with-auto-resistance-too/), wondering if the same can be done for the Zwift Hub (which was unfortunately discontinued).

**Comment by cagnulein** [comment]
yes it's supported! are you on android or ios? Probably I added this in the beta
Let me know!

**Reply by bdattilo** [reply]
I am on Android. Would be great to try this out!

**Comment by cagnulein** [comment]
Great download the app from the store and then join the beta from this
https://play.google.com/apps/testing/org.cagnulen.qdomyoszwift

Let me know if you have any issue!

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno ven 9 feb 2024 alle 18:32 bdattilo ***@***.***> ha
scritto:

**Reply by bdattilo** [reply]
Thank you for the link, and sorry for my time to respond!

I think it is working - I can connect to the Zwift Hub and I do feel the resistance changing due to the Peloton class. However, the auto resistance seems much too hard compared to a previous spin bike. is there any settings that can be used to tweak how strong the resistance is?

Thanks again for the great app!

**Reply by cagnulein** [reply]
@bdattilo hi use the peloton gain under the peloton settings to lower it down. for example put a 1.5

let me know
